analysis/wwz/wwz4l.py

```diff
@@ -482,9 +490,8 @@ def process(self, events):
             veto_map_mask = (veto_map_array == 0)
 
             ##### JME Stuff #####
```

Comment:
Could you modify the start and end comments to be harmonized? 

E.g. `##### Start of JME Stuff #####` and `##### End of JME Stuff #####` or something. Just something to make it more clear that they go together.